### PR TITLE
Fixed Carousel on smaller screen

### PR DESCRIPTION
**Pull request by Ayusht777**
#20 
Screenshot (293)

### CODE REVIEW

File: src/app/components/carousel.tsx [hunk 1]
```diff
@@ -73,10 +73,7 @@ const Carousel = () => {
               size="large"
             ></Button>
           </div>
-          <div
-            className={styles.flexcontainer}
```
Comment (fliptrail) [inline src/app/components/carousel.tsx:77]: Don't remove original flexcontainer class. Instead add another class here.

File: src/app/components/carousel.tsx [hunk 2]
```diff
@@ -73,10 +73,7 @@ const Carousel = () => {
...
-            style={{ flexDirection: "column", margin: "20px" }}
-          >
+          <div className={styles.CarouselImgContainer}>
```
Comment (fliptrail) [inline src/app/components/carousel.tsx:76]: Remove

File: src/app/page.module.css [hunk 3]
```diff
@@ -9,3 +9,15 @@
...
   margin-bottom: 4px;
 }
+
+.CarouselImgContainer {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  margin: 20px;
+}
+@media (max-width: 390px) {
```
Comment (fliptrail) [inline src/app/page.module.css:19]: Use breakpoints in accordance with Fluent 2 guidelines. Ref: https://fluent2.microsoft.design/layout#responsive-design

**Review by fliptrail — CHANGES_REQUESTED** [review]

**Comment by fliptrail** [comment]
@Ayusht777 I've marked up comments.

**Comment by Ayusht777** [comment]
check the code now

**Comment by fliptrail** [comment]
@Ayusht777 npm build is failing
```

Type error: Spread types may only be created from object types.
--
14:42:27.660 |  
14:42:27.660 | 76 \|           <div
14:42:27.660 | 77 \|             className={{
14:42:27.660 | > 78 \|               ...styles.flexcontainer,
14:42:27.660 | \|               ^
14:42:27.660 | 79 \|               ...styles.CarouselImgContainer,
14:42:27.660 | 80 \|             }}
14:42:27.660 | 81 \|           >
14:42:27.775 | Error: Command "npm run build" exited with 1

```

**Pull request closed without merging**